site/docs/v1/tech/email-templates/index.adoc

```diff
@@ -36,6 +36,27 @@ Here are the topics in this section:
 
 * link:/docs/v1/tech/email-templates/message-templates/[Message Templates]
 
+== Common Providers
```

Comment:
SMTP Providers? This could either go here, or in the Tenant Email configuration section?